src/auth/strategies/steam.strategy.spec.ts

```diff
@@ -5,14 +5,12 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { SteamStrategy } from './steam.strategy';
 import { Error } from 'mongoose';
 
-const mockPlayer: Player = {
-  id: 'FAKE_PLAYER_ID',
-  name: 'FAKE_PLAYER',
-  steamId: 'FAKE_STEAM_ID',
-  hasAcceptedRules: true,
-  roles: [],
-  _links: [],
-};
+const mockPlayer = new Player();
+mockPlayer.id = 'FAKE_PLAYER_ID';
+mockPlayer.name = 'FAKE_PLAYER';
+mockPlayer.steamId = 'FAKE_STEAM_ID';
+mockPlayer.hasAcceptedRules = true;
+mockPlayer.roles = [];
```

Comment:
Why not define these properties within the `Player()` constructor instead of modifying this?

Reply:
Hmm, dunno... Good idea, though

Reply:
can be done in a follow up